engine/render/renderer/renderer/Renderer2D.cpp

```diff
@@ -92,4 +92,6 @@ void Renderer2D::Flush()
     models.Clear();
     transforms.Clear();
 }
+
+void Renderer2D::Destroy() {}
```

Comment:
Generally default anything that's unimplemented so it's easier to read.

Reply:
Removed.